android/src/main/java/com/expensify/livemarkdown/MarkdownBlockquoteSpan.java

```diff
@@ -36,10 +38,14 @@ public void drawLeadingMargin(Canvas c, Paint p, int x, int dir, int top, int ba
 
     p.setStyle(Paint.Style.FILL);
     p.setColor(borderColor);
+    for(int stripe = 0; stripe < nestLevel; stripe++) {
```

Comment:
Let's add a space between `for` and `(` and maybe rename the variable?
```suggestion
    for (int level = 0; level < nestLevel; level++) {
```